tox.ini

```diff
@@ -23,4 +23,4 @@ commands =
 deps = pip-tools
 commands =
   pip-compile --upgrade --no-header --no-emit-options --strip-extras --output-file requirements.txt requirements.in
-  python -c 'import shutil, os; os.path.exists(".tox") and shutil.rmtree(".tox")'
```

Comment:
[it seem](https://github.com/NetherlandsForensicInstitute/hansken-extraction-plugin-template-python/pull/10#discussion_r1768198185)

This is probably not the way, but let's only remove the `py3` venv, not the `upgrade` venv. (more modern tox versions indeed failed by simply deleting the `.tox` file)